client/lib/olark/index.js

```diff
@@ -368,8 +372,28 @@ const olark = {
 			this.userType = userType;
 			return true;
 		} );
+	},
+
+	notifyLocation( location ) {
+		const { details } = olarkStore.get();
```

Comment:
you can cut out a few lines here and you won't run into exceptions if these don't exist because of the soft-fail modes of destructuring

```js
const { details: { isConversing = false } } = olarkStore.get();

if ( ! isConversing ) {
	return;
}
```